$NetBSD: patch-js_moz.configure,v 1.1 2020/12/05 12:03:26 nia Exp $

Check ${PREFIX} to allow use of non-builtin readlink libraries.

--- js/moz.configure.orig	2020-11-04 10:51:54.000000000 +0000
+++ js/moz.configure
@@ -696,7 +696,7 @@ def editline(js_shell, is_windows, compi
 js_option('--enable-readline', help='Link js shell to system readline library',
           when=editline)
 
-has_readline = check_symbol('readline', flags=['-lreadline'], when='--enable-readline',
+has_readline = check_symbol('readline', flags=['-lreadline', '-L@PREFIX@/lib'], when='--enable-readline',
                             onerror=lambda: die('No system readline library found'))
 
 set_config('EDITLINE_LIBS', ['-lreadline'], when=has_readline)
